lib/docker.js

```diff
@@ -331,10 +331,16 @@ function ensureDockerVersion() {
         }
 
         var minimumDockerVersion = os.type() === 'Darwin' ? '1.12.0' : '1.8.0';
-        if (semver.lt(dockerVersion, minimumDockerVersion)) {
-            console.error('Building the deploy repo on ' + os.type() +
-                ' supported only with docker ' + minimumDockerVersion + '+');
-            process.exit(1);
+        try {
+            if (semver.lt(dockerVersion, minimumDockerVersion)) {
```

Comment:
Perhaps better `dockerVersion = dockerVersion.replace(/\.0+/g, '.')` to get rid of the 0-padding but leave the check in place?